tests_e2e/cypress/integration/main.js

```diff
@@ -56,7 +56,7 @@ describe('First Test', () => {
         .should('have.attr', 'value', 'Start').click();
 
       // Minimal check that JupyterLab has opened
-      cy.get('div#jp-MainLogo', { timeout: 60000 }).should('exist').wait(2000);
+      cy.get('div#jp-MainLogo', { timeout: 60000 }).should('exist').wait(10000);
```

Comment:
Do we need this? Seems unrelated to the PR.

Reply:
This has been changed in main now